lib/src/core/models/frame_model.dart

```diff
@@ -0,0 +1,20 @@
+import 'package:json_annotation/json_annotation.dart';
+
+import 'location_model.dart';
+
+part 'frame_model.g.dart';
+
+@JsonSerializable(explicitToJson: true)
+class FrameModel {
+  FrameModel({
+    this.house_details,
+    this.frames,
+  });
+
+  factory FrameModel.fromJson(Map<String, dynamic> json) =>
+      _$FrameModelFromJson(json);
+  Map<String, dynamic> toJson() => _$FrameModelToJson(this);
```

Comment:
In case, we want to sort constructors first, then we shouldn't touch the methods and keep them after fields and properties